examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,102 @@
+## Multi-Tenant logging using Fluent-bit, EKS , Amazon MSK and  Amazon OpenSearch
+
+In this example, we will showcase how to send your application logs from EKS to OpenSearch via Amazon MSK (managed KAFKA). We will use tenant notion using EKS "namespace" such that each tenant has a seperate namespace.  The solution helps to meet following use cases in a multi-tenant deployment from EKS cluster.
+
+* Use of a broker to de-couple sending logs directly to OpenSearch so that broker can store logs if the OpenSearch is not available or having  some issues. In this example we will use Amazon MSK to store our logs.
```

Comment:
I think we need an intro for what a broker is. This sort of thing may be should have a short terminology section at the beginning introducing how all terms are used.

Reply:
> having  some issues.

extra space